docs/features/vision/fog_of_war_manager.md

```
@@ -0,0 +1,433 @@
+	- onready
+    - value = find_child("ScreenOverlay")
+- \_visibility_field
+	- type  Area3D
+	- onready
+    - value = find_child("VisibilityField")
+
+## **Events**
+
+- None
+
+## **Dependencies**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)